cmd/authgear/server/server.go

```diff
@@ -54,7 +55,7 @@ func (c *Controller) Start() {
 		c.logger.Warn("development mode is ON - do not use in production")
 	}
 
-	configSrcController := newConfigSourceController(p)
+	configSrcController := newConfigSourceController(p, context.Background())
```

Comment:
@louischan-oursky Would like to ask, I am not sure if it is appreciate to use the background context here? The context is for the `SQLExecutor` of config source store. Any idea?

Reply:
The main usage of the context is to provide cancellation. In case it is a HTTP request, the context is the request context, which is cancelled if the connection closes. If we use background context here, then the SQLExecutor may hang indefinitely because the context never cancel itself.

Reply:
But I guess it is fine to use background context here.